.github/workflows/zulip-ci.yml

```diff
@@ -18,19 +18,14 @@ jobs:
         include:
           # Base images are built using `tools/ci/Dockerfile.prod.template`.
           # The comments at the top explain how to build and upload these images.
-          # Debian 10 ships with Python 3.7.3.
-          - docker_image: zulip/ci:buster
-            name: Debian 10 Buster (Python 3.7, backend + frontend)
-            os: buster
-            include_frontend_tests: true
```

Comment:
We probably want to integrate https://github.com/zulip/zulip/pull/21659 together with this, since we're changing the set of jobs required to pass.